projects/examples/src/components/formly/input/formly-inputs.example.component.ts

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component } from '@angular/core';
+import { FormlyFieldConfig, FormlyFormOptions } from '@ngx-formly/core';
+import { Unit, VCD_FORMLY_INPUT_TYPES, VcdFormlyFieldConfig } from '@vcd/ui-components';
+import { Bytes } from '../../../../../components/src/utils';
+
+enum UI_FIELDS {
+    vcdFormlyInput = 'vcdFormlyInput',
+    vcdFormlySelect = 'vcdFormlySelect',
+    vcdNumberWithUnitInput = 'vcdNumberWithUnitInput',
+}
+
+@Component({
+    template: `
+        <div class="clr-form-horizontal">
+            <formly-form [model]="model" [fields]="fields" [options]="options"></formly-form>
```

Comment:
Please add a high level comment explaining what example is and how it works.

A sentence or two in the component doc and more words in the HTML so we can understand the example without reading the code. It's not clear to me what this is an example of? The fact that we have wrappers? I think this example needs quite a bit of documentation. 

Is this going to be a single example for all inputs? Probably shouldn't each example is showing one feature